css/app/datepicker.css

```diff
@@ -89,7 +89,16 @@
 	border-left: 0;
 }
 
+.todayHilight {
+	background-color: rgb(255, 255, 170);
+}
+
+#datepicker table tbody tr.uib-weeks td:first-child button {
+	background-color: rgba(240, 240, 240, .9);
+}
+
 #datepicker table tbody tr.uib-weeks td:last-child button {
+	background-color: rgba(240, 240, 240, .9);
```

Comment:
I didn't test this yet, but I think this will only work for US. What if you set your locale e.g. to german, where the week starts on monday? I guess Monday and Sunday will be shown a little darker